packages/accordions/src/elements/stepper/Stepper.tsx

```diff
@@ -41,7 +41,7 @@ interface IStepperProps extends HTMLAttributes<HTMLOListElement> {
  *  - `Stepper.Label`
  *  - `Stepper.Content`
  * 
- * @extends HTMLOListElement
+ * @extends OlHTMLAttributes<HTMLOListElement>
```

Comment:
Should we update the actual type definition as well to reflect this? (unrelated to documentation, but this feels like a good move overall)

```
interface IStepperProps extends OlHTMLAttributes<HTMLOListElement> {
   ...
}
```

Reply:
The extends have all been removed for the time being. cc @jzempel